New ships & outfits for the Unfettered (#7019)
endless-sky/endless-sky: data/hai/hai ships.txt

		"Hyperdrive"

ship "Sea Scorpion"
	sprite "ship/hai sea scorpion"
	thumbnail "thumbnail/hai sea scorpion"
	attributes
		category "Medium Warship"
		"cost" 5450000
		"shields" 15500
		"hull" 4900
		"required crew" 14
		"bunks" 36
		"mass" 360

> 		"drag" 4.7

Reviewer: ```suggestion
		"drag" 5.2
```

		"heat dissipation" .7
		"fuel capacity" 500
		"cargo space" 45

> 		"outfit space" 510
> 		"weapon capacity" 168
> 		"engine capacity" 104

Reviewer: ```suggestion
		"outfit space" 525
		"weapon capacity" 168
		"engine capacity" 124
```

		weapon
			"blast radius" 84
			"shield damage" 840
			"hull damage" 420
			"hit force" 1260
	outfits
		"Hai Tracker Pod" 4
		"Chameleon Anti-Missile" 2
		"Tracker Storage Pod" 4
		"Hai Tracker" 392
		"Hai Williwaw Cooling" 2
		"Boulder Reactor"
		"Hai Ravine Batteries"
		"Hai Diamond Regenerator"
		`"Bondir" Atomic Thruster`

> 		`"Biroo" Atomic Steering`

Reviewer: ```suggestion
		`"Bondir" Atomic Steering`
```

		"Hyperdrive"
		"Pulse Rifle" 15

	engine -28.5 68.5
	engine 28.5 68.5
	"steering engine" 64.5 23.5
		zoom 0.8

> 		angle -90

Reviewer 2: ```suggestion
		angle 90
```

> 		left

Reviewer 2: ```suggestion
		right
```

	"steering engine" -64.5 23.5
		zoom 0.8

> 		angle 90

Reviewer 2: ```suggestion
		angle -90
```

> 		right

Reviewer 2: ```suggestion
		left
```

	gun -13 -77.5
	gun 13 -77.5
	gun -27 -56
	gun 27 -56
	turret -35.5 24.5
	turret 35.5 24.5
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"

> 	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are able to mass produce them, enabling them to employ new tactics against the other Hai, alleviating some of their numerical, technological, and logistical disadvantages."

Reviewer 3: ```suggestion
	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are finally able to mass produce them. The ever growing fleet of Sea Scorpions enables the Unfettered to employ new tactics against the other Hai which alleviate some of their numerical, technological, and logistical disadvantages."
```

> 	description "Most Unfettered ships aren't usable as missile boats, so they previously relied on pulse weapons instead. Only after several decades of hard work were they able to take the Sea Scorpion into mass production, but it was well worth it; its low price, agility and large number of gun ports make it perfect for its role, even at the cost of its weak hull and shielding."

Reviewer: ```suggestion
	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are able to mass produce them, enabling them to employ new tactics against the other Hai, alleviating some of their numerical, technological, and logistical disadvantages."
```

			"hull damage" 420
			"hit force" 1260
	outfits
		"Hai Tracker Pod" 4
		"Chameleon Anti-Missile" 2
		"Tracker Storage Pod" 4
		"Hai Tracker" 392
		"Hai Williwaw Cooling" 2
		"Boulder Reactor"
		"Hai Ravine Batteries"
		"Hai Diamond Regenerator"
		`"Bondir" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Value Detector"
		"Hyperdrive"
		"Pulse Rifle" 15

	engine -28.5 68.5
	engine 28.5 68.5
	"steering engine" 64.5 23.5
		zoom 0.6
		angle -90
		right
	"steering engine" -64.5 23.5
		zoom 0.6
		angle 90
		left
	gun -13 -77.5
	gun 13 -77.5
	gun -27 -56
	gun 27 -56
	turret -35.5 24.5
	turret 35.5 24.5
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"
	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are finally able to mass produce them. The ever growing fleet of Sea Scorpions enables the Unfettered to employ new tactics against the other Hai which alleviate some of their numerical, technological, and logistical disadvantages."

> 	description "	One single Sea Scorpion may not hope to do much more than use its mobility to stay clear of the enemy's weapons, but a pack of them is able to strike at bigger targets, covering each other and retreating to regenerate their shields if needed, always striking in the enemy's blind spots until they can finally come in for the kill."

Author: Well all trackers is alone, I'll specify that as a pack they use shorter range weapons or overwhelm AM

> 	description "	One single Sea Scorpion may not hope to do much more than use its mobility to stay clear of the enemy's weapons, but a pack of them is able to strike at bigger targets, covering each other and retreating to regenerate their shields if needed, always striking in the enemy's blind spots until they can finally come in for the kill."

Author: How is it now?

> 	description "	One single Sea Scorpion may not hope to do much more than use its mobility to stay clear of the enemy's weapons, but a pack of them is able to strike at bigger targets, covering each other and retreating to regenerate their shields if needed, always striking in the enemy's blind spots until they can finally come in for the kill."

Reviewer 4: All concerns addressed. :)

ship "Sea Scorpion"
	sprite "ship/hai sea scorpion"
	thumbnail "thumbnail/hai sea scorpion"
	attributes
		category "Medium Warship"
		"cost" 5450000
		"shields" 15500
		"hull" 4900
		"required crew" 14
		"bunks" 36
		"mass" 360
		"drag" 5.2
		"heat dissipation" .7
		"fuel capacity" 500
		"cargo space" 45
		"outfit space" 510
		"weapon capacity" 168
		"engine capacity" 117
		weapon
			"blast radius" 84
			"shield damage" 840
			"hull damage" 420
			"hit force" 1260
	outfits
		"Hai Tracker Pod" 4
		"Chameleon Anti-Missile" 2
		"Tracker Storage Pod" 4
		"Hai Tracker" 392
		"Hai Williwaw Cooling" 2
		"Boulder Reactor"
		"Hai Ravine Batteries"
		"Hai Diamond Regenerator"
		`"Bondir" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Value Detector"
		"Hyperdrive"
		"Pulse Rifle" 15

	engine -28.5 68.5

> 		zoom 0.65

Author: It looked too big to my taste otherwise since its a relatively small ship
I could tune it down a bit though

> 		zoom 0.65

Reviewer 5: Then that sounds like the ship has too much engine space, or has a sprite that is too small.

> 		zoom 0.65

Author: Its meant to be a very mobile ship
It looks fine in normal mode its not that

I'll be honest I have no idea how engines work

	engine 28.5 68.5
		zoom 0.65
	"steering engine" -28.5 68.5
		zoom 0.35
		right
	"steering engine" 28.5 68.5
		zoom 0.35
		left
	"steering engine" 64.5 23.5
		zoom 0.5
		angle -90
		right
	"steering engine" -64.5 23.5
		zoom 0.5
		angle 90
		left
	gun -13 -77.5
	gun 13 -77.5
	gun -27 -56
	gun 27 -56
	turret -35.5 24.5
	turret 35.5 24.5
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"
	description "The Sea Scorpion was designed to serve as a mobile platform to launch Trackers from. After decades of hard work, the Unfettered are finally able to mass produce them. The ever growing fleet of Sea Scorpions enables the Unfettered to employ new tactics against the other Hai which alleviate some of their numerical, technological, and logistical disadvantages."

> 	description "	One single Sea Scorpion may not hope to do much more than use its mobility to stay clear of the enemy's weapons, using trackers and utilizing its range advantage. A pack of them could easily overwhelm any amount of anti missile put against them. However, where the Sea Scorpion really shine is in small hunting groups, when equipped with pulse weapons. Together, they are able to strike at isolated bigger targets, covering each other and retreating to regenerate their shields if needed, always striking in the enemy's blind spots until they can finally come in for the kill, and plunder."

Reviewer 5: Same as the outfits; you don't need to go into exact detail about the specifics on how the Unfettered pilot their ships.
```suggestion
```

Author: It adds depht to the game, I like to have a bit of lore hidden in descriptions

Reviewer 5: This isn't lore, this is a basic description of swarm tactics that is not even necessarily exclusive to the Unfettered.

Author: Its a story about those ships, how they're meant to work; a living description of sorts
But if removing it is required I'll do it
